app/lib/features/chat/widgets/avatar_builder.dart

```diff
@@ -26,6 +27,21 @@ class AvatarBuilder extends ConsumerWidget {
         return Padding(
           padding: const EdgeInsets.only(right: 10),
           child: ActerAvatar(
+            onAvatarTap: () async {
+              final member = await ref.read(
+                memberProvider(
+                  (userId: userId, roomId: roomId),
+                ).future,
+              );
+              // ignore: use_build_context_synchronously
+              showMemberInfoDrawer(
+                context: context,
+                memberProfile: profile,
+                roomId: roomId,
+                member: member!,
+                memberId: userId,
+              );
```

Comment:
Do we require all this params here? Can't we just pass opponent `userId` and get rest of the informations in `_MemberInfoDrawer` using require providers?

Reply:
Thanks for details.
Understood your point. Good for now. :)